source/features/notifications-filter.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import {CheckIcon, CheckCircleIcon, XCircleIcon, IssueOpenedIcon, GitPullRequestIcon, GitPullRequestDraftIcon, GitMergeIcon, DotIcon, DotFillIcon} from '@primer/octicons-react';
+
+import features from '.';
+
+const filters = {
+	PR: '[href*="/pull/"]',
+	Issue: '[href*="/issues/"]',
+	Open: ':is(.octicon-issue-opened, .octicon-git-pull-request)',
+	Closed: ':is(.octicon-issue-closed, .octicon-git-pull-request-closed)',
+	Draft: '.octicon-git-pull-request-draft',
+	Merged: '.octicon-git-merge',
+	Read: 'notification-read',
+	Unread: 'notification-unread'
```

Comment:
Are these selectors or just class names? Instead of using `.classList.contains()` just use `.matches()` so you can turn this into regular selectors like the rest of them.